Pipelines/cli/devskim-cli-pr.yml

```diff
@@ -7,19 +7,14 @@ pr:
   branches:
     include:
     - main
-  paths:
-    include:
-    - Pipelines/cli/devskim-cli-pr.yml
-    - DevSkim-DotNet
-    - rules
-    - DevSkim-VSCode-Plugin
 
 resources:
   repositories:
     - repository: templates
       type: git
       name: SecurityEngineering/OSS-Tools-Pipeline-Templates
-      ref: refs/tags/v1.0.5
+      #ref: refs/tags/v1.0.5
+      ref: da/add-retry-to-notice
```

Comment:
Is this a permanent branch?

Reply:
One of the pipeline tasks is flakey so I've added a retry to the template, but testing to make sure it works first. Will update to a tag reference once this run completes.